Common.mk

```diff
@@ -26,7 +26,7 @@ OUTPUT_BIN_DIR?=$(OUTPUT_DIR)/bin/$(REPO)
 #################### AWS ###########################
 AWS_REGION?=us-west-2
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
-ARTIFACT_BUCKET?=my-s3-bucket # TODO: in eks-a we use ARTIFACTS_BUCKET
+ARTIFACTS_BUCKET?=my-s3-bucket
```

Comment:
is there any place, such as in the Prow jobs, where we're over-riding this variable using the `ARTIFACT_BUCKET` name?

Reply:
Yes -- loads. Good catch! I'll update there

Reply:
Updated to avoid changing in so many places at once